podpac/core/data/file_source.py

```diff
@@ -106,6 +106,10 @@ def dataset(self):
         if self.source.startswith("s3://"):
             _logger.info("Loading AWS resource: %s" % self.source)
             self._file = self.s3.open(self.source, "rb")
+            import pdb
+
+            pdb.set_trace()  # breakpoint 0dc1960c //
+
```

Comment:
I'm quite confused -- it looks like my local copy is up-to-date but I don't have the above breakpoint in my file_source.py, so I can't remove these lines. @jmilloy mind having a look?